.github/stale.yml

```diff
@@ -1,62 +1,50 @@
 # Configuration for probot-stale - https://github.com/probot/stale
 
-# Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 30
-
-# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
-# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-daysUntilClose: 7
-
 # Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
 onlyLabels: []
 
 # Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable.
 # We want stale bot to notify us that something is stale so we can revisit it.
 # If one issue is marked as 'reminder' by the reminder bot, we don't mark it as 'stale' again.
 exemptLabels:
-  - reminder
+  - remindme
```

Comment:
Unfortunately we cannot update the label here. The reminder bot uses a hardcode label here https://github.com/probot/reminders/blob/7dc50c529eb2b8fa06696a3b4e16769ee841ad0c/lib/reminders.js#L5

Reply:
cool thx